.github/workflows/post-merge.yaml

```diff
@@ -57,10 +57,57 @@ jobs:
         run: make ci-go-test-coverage
         timeout-minutes: 30
 
+  release-build:
```

Comment:
https://docs.github.com/en/actions/creating-actions/creating-a-composite-run-steps-action

This will let us de-duplicate this _a lot_. Creating a composite action, I believe we should be able to have one action used in the three workflows, with the inputs:

a. golang version (just because I'd like this to be visible at the "action call site"
b. telemetry URL (because it differs between the workflows)

(Not today, but before this merges.)

Reply:
No way. Composite actions cannot use the "uses" machinery, only shell actions. There's not much to gain then. Let's refractor when they're fully capable.